engine/crates/validation/tests/validation_errors/unknown_type_in_schema_def.errors.txt

```diff
@@ -1,20 +1,20 @@
-  × Cannot set schema query root to unknown type type `Dunno`
+  × Cannot set schema query root to unknown type `Dunno`
    ╭─[1:1]
  1 │ schema {
    · ──────
  2 │     query: Dunno
    ╰────
```

Comment:
Unrelated to this PR, but the span we've chosen for this error message is a bit confusing - should we not be underlining `Dunno`?  (Potentially with a label if we wanted to make it even more helpful)

Reply:
True, opening a linear issue to do it when there is time for a small issue

Reply:
Ah I think I maybe remember that these don't have spans in the async graphql parser AST. But I'll double check.

Reply:
That would explain it if so - odd choice from `async_graphql` though, I usually remember having to do `.node` about 1000 times to get through all the `Positioned<T>` it uses

Reply:
I know the pain :p